cassini_v2_files/metadata-reference.py

```
@@ -0,0 +1,59 @@
+    re.DOTALL | re.VERBOSE,
+)
+
+def remove_references_preserve_format(xml_file: Path) -> int:
+    """
+    Remove matching <Internal_Reference> blocks from one file.
+    Returns the number of blocks removed.
+    """
+    text = xml_file.read_text(encoding="utf-8")
+    count = 0
+
+    def _count_and_remove(_m):
```

Ruff (0.15.6)

[warning] Missing return type annotation for private function `_count_and_remove`

Add return type annotation: `str`

(ANN202)